### ISSUE TITLE
Very low validation accuracy using imagenet_classification.py

### ISSUE THREAD

**Issue by xlvy** [tensorflow/tensorrt#220]
Run environment:

- docker version: nvcr.io/nvidia/tensorflow:20.01-tf2-py3 with tensorflow version 2.0 and TensorRT 7.0

- Validation TFRecord files:  [https://github.com/tensorflow/models/blob/master/research/slim/datasets/build_imagenet_data.py](url)

- Model: TF2.0 Resnet_50 hub model [https://tfhub.dev/tensorflow/resnet_50/classification/1](url)

- preprocess_method: vgg

Why is the accuracy only 0.1%? Is there something wrong?
![image](https://user-images.githubusercontent.com/51997291/97069368-74c8e280-1602-11eb-83d5-ae2398040589.png)
